frontend/src/store/modules/auth.js

```diff
@@ -1,42 +1,97 @@
 import axios from 'axios';
 
+import { interceptorRefresh } from '../../main.js';
+
 export default {
   namespaced: true,
   state: {
-    token: localStorage.getItem('token') || ''
+    token: localStorage.getItem('token') || '',
```

Comment:
What are requirements about browsers? I don't know much about Vue build process, but maybe (just maybe) we could use that shiny `localStorage.getItem('token') ?? ''` null coalescing syntax

Reply:
From docs it looks superb, but as You mentioned browser support is quite poorly, so I am not sure. @OtisRed what do You think ?

Reply:
It's over my expertise to say sth meaningful here